README.md

```diff
@@ -2,6 +2,15 @@
 
 [See the Storybook](https://v1--613a8e945a5665003a05113b.chromatic.com/)
 
+# How to install?
+
+To run the project, first You need to use a the tool for managing the node version, which is convenient for you.
+We suggest using [Volta](https://volta.sh/).
+
+Required version of node is `16.13.2` and npm `8.1.0`.
```

Comment:
how I can easily switch to this version? Is there some file like `.nvmrc` with the node version so I can switch using one command like `nvm use` when using nvm?

Reply:
We've configured the project to use volta, we could also introduce `.nvmrc` file if you think it's necessary.

Reply:
ok, I just read the documentation, and I discovered that node version is changed automatically, so there is no need to do something like `nvm use` every time you change the project 👍 
I think it's worth mentioning that